src/eth/rpc.ts

```diff
@@ -19,6 +19,18 @@ export const getEarnings = async(address:string, endRound: BigNumber): Promise<{
     }
 }
 
+const getDelegatorSnapshot = async (
+    delegator: string,
+    delegate: string,
+    snapshotRound: BigNumber,
+) => {
```

Comment:
skipped explicit return type to keep fn short, as it is auto-inferred from `getEarnings` and params